manifests/profile/mistral.pp

```diff
@@ -359,6 +367,20 @@
           content => template('st2/etc/init/mistral.conf.erb'),
           notify  => Service['mistral'],
         }
+        if $api_service {
```

Comment:
yes: in line 342 it sets the flag undef it off when `api_service` is true, so service list won't have api (tested).